### PR TITLE
feat(26270): Add Skeleton UI for Money Requests

### PR DESCRIPTION
**Pull request by mananjadhav**
### Details

- Adds the `SkeletonView` to the Money Requests, while the transaction is being loaded.

### Fixed Issues

$ https://github.com/Expensify/App/issues/26270
PROPOSAL: 

### Tests

- [x] Check the skeleton UI is loaded for the request money
- [x] Check the skeleton UI is loaded for the distance money request
- [x] Verify the skeleton UI design is as per this [spec](https://user-images.githubusercontent.com/1127863/264535814-07f647d8-2a95-4c7d-af29-2ac67378153e.png).
- [x] Verify the skeleton UI is hidden once the transaction is loaded.
- [x] Verify that no errors appear in the JS console

### Offline tests

- Skeleton UI won't be loaded in the offline mode, as we see greyed out amounts for the transaction.

### QA Steps

**For all platforms**

1. Open the app, logout if you're already logged in. (this is to ensure that we don't load the transactions from local storage).
2. You can enable throttling with slow 3G/2G for the longer duration on the SkeletionUI.
3. Go to any chat where you have an existing IOU (Money Requests)
4. Verify that the skeleton view is showed for the money preview.
5. Once the transaction is loaded, it shows the amount, and description.
6. Repeat steps 3 to 5 for any distance requests. It should show skeleton view below the map.
7. Verify the skeleton view, looks like the following:

image

- [ ] Verify that no errors appear in the JS console

### PR Author Checklist

- [x] I linked the correct issue in the `### Fixed Issues` section above
- [x] I wrote clear testing steps that cover the changes made in this PR
    - [x] I added steps for local testing in the `Tests` section
    - [x] I added steps for the expected offline behavior in the `Offline steps` section
    - [x] I added steps for Staging and/or Production testing in the `QA steps` section
    - [x] I added steps to cover failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
    - [x] I tested this PR with a [High Traffic account](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#high-traffic-accounts) against the staging or production API to ensure there are no regressions (e.g. long loading states that impact usability).
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I ran the tests on **all platforms** & verified they passed on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] I verified there are no console errors (if there's a console error not related to the PR, report it or open an issue for it to be fixed)
- [x] I followed proper code patterns (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`)
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
      - [x] If any non-english text was added/modified, I verified the translation was requested/reviewed in #expensify-open-source and it was approved by an internal Expensify engineer. Link to Slack message:
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I followed the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I tested other components that can be impacted by my changes (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` are working as expected)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] I verified that if a function's arguments changed that all usages have also been updated correctly
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] If we are not using the full Onyx data that we loaded, I've added the proper selector in order to ensure the component only re-renders when the data it is using changes
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG)`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If the PR modifies a component or page that can be accessed by a direct deeplink, I verified that the code functions as expected when the deeplink is used - from a logged in and logged out account.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR author checklist, including those that don't apply to this PR.

### Screenshots/Videos
<details>
<summary>Web</summary>

**Distance Requests**

https://github.com/Expensify/App/assets/3069065/2bbd7e39-cf15-46ad-87bf-38e04846c46e

**Multiple Requests**

https://github.com/Expensify/App/assets/3069065/9737f514-b7f7-48aa-83ee-3376c1be6c8a

**IOU Requests**

https://github.com/Expensify/App/assets/3069065/8f5cc6ef-b915-4b00-b828-73158fc6f145

</details>

<details>
<summary>Mobile Web - Chrome</summary>

https://github.com/Expensify/App/assets/3069065/708df8f3-995e-405b-b14e-c8a78d0cf22b

</details>

<details>
<summary>Mobile Web - Safari</summary>

https://github.com/Expensify/App/assets/3069065/3ff8be1b-a19a-4be6-b498-766a423fc87c

</details>

<details>
<summary>Desktop</summary>

https://github.com/Expensify/App/assets/3069065/10468911-2dcf-42b2-8046-2e57bee5a836

</details>

<details>
<summary>iOS</summary>

https://github.com/Expensify/App/assets/3069065/950304c3-1fbc-48e2-ba94-1c7f7825ed12

</details>

<details>
<summary>Android</summary>

https://github.com/Expensify/App/assets/3069065/8d4ead57-ceae-4fb0-81ea-5e6f3eacc114

</details>

### CODE REVIEW

**Comment by hayata-suenaga** [comment]
@mananjadhav Is it possible that you can push the rest of the changes soon? I think we want to fix this as soon as possible 🙇 (i.e. we want to deploy this to production)

**Comment by mananjadhav** [comment]
I am just working on the checklist.

**Comment by mananjadhav** [comment]
@hayata-suenaga @mountiny @shawnborton @fedirjh The PR is ready.

**Review by parasharrajat — CHANGES_REQUESTED** [review]

**Comment by mountiny** [comment]
niiice, videos looking good!

File: src/components/MoneyRequestSkeletonView.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import withWindowDimensions, {windowDimensionsPropTypes} from './withWindowDimensions';
+import variables from '../styles/variables';
+import themeColors from '../styles/themes/default';
+import compose from '../libs/compose';
+import withLocalize, {withLocalizePropTypes} from './withLocalize';
+
+const propTypes = {
+    ...windowDimensionsPropTypes,
+    ...withLocalizePropTypes,
+    shouldAnimate: PropTypes.bool,
```
[the remark this answers is not in the document]
Comment (mananjadhav) [reply src/components/MoneyRequestSkeletonView.js]: Don't need any of these props.

**Comment by mananjadhav** [comment]
@parasharrajat I've fixed the comments. Should've raised a PR fresh in the morning to avoid silly changes. Thanks for catching those.

File: src/components/ReportActionItem/MoneyRequestPreview.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import {View} from 'react-native';
 import PropTypes from 'prop-types';
 import {withOnyx} from 'react-native-onyx';
 import lodashGet from 'lodash/get';
+import lodashIsEmpty from 'lodash/isEmpty';
 import _ from 'underscore';
```
[the remark this answers is not in the document]
Comment (mananjadhav) [reply src/components/ReportActionItem/MoneyRequestPreview.js]: Fixed.

File: src/components/MoneyRequestSkeletonView.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+const defaultProps = {
+    shouldAnimate: true,
+};
+
+function MoneyRequestSkeletonView(props) {
+    return (
+        <View style={[styles.flex1, styles.overflowHidden]}>
+            <SkeletonViewContentLoader
+                animate={props.shouldAnimate}
+                width={styles.w100.width}
```
[the remark this answers is not in the document]
Comment (mananjadhav) [reply src/components/MoneyRequestSkeletonView.js]: I had issues with the skeleton UI overflowing, but I fixed with fixed values. Can be removed.

File: src/components/MoneyRequestSkeletonView.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+                />
+            </SkeletonViewContentLoader>
+        </View>
+    );
+}
+
+MoneyRequestSkeletonView.propTypes = propTypes;
+MoneyRequestSkeletonView.defaultProps = defaultProps;
+MoneyRequestSkeletonView.displayName = 'IOUSkeletonView';
+export default compose(withWindowDimensions, withLocalize)(MoneyRequestSkeletonView);
```
[the remark this answers is not in the document]
Comment (mananjadhav) [reply src/components/MoneyRequestSkeletonView.js]: Removed.

File: src/components/MoneyRequestSkeletonView.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+                    height="8"
+                />
+            </SkeletonViewContentLoader>
+        </View>
+    );
+}
+
+MoneyRequestSkeletonView.propTypes = propTypes;
+MoneyRequestSkeletonView.defaultProps = defaultProps;
+MoneyRequestSkeletonView.displayName = 'IOUSkeletonView';
```
[the remark this answers is not in the document]
Comment (mananjadhav) [reply src/components/MoneyRequestSkeletonView.js]: yeah missed this one.

File: src/components/MoneyRequestSkeletonView.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (mananjadhav) [reply src/components/MoneyRequestSkeletonView.js]: Removed all the props.

**Comment by mananjadhav** [comment]
@parasharrajat I've updated the description so that `Web` attachments show up.

> We can work on it in the IST zone in the morning to if there are more requests.

Sure that'll be great!

**Comment by mananjadhav** [comment]
Yeah I thought so too. I am only replacing with the skeleton view, the behavior exists for other wise IOU requests too. I don't have the message context too that I can replace the message with any other skeleton view.


https://github.com/Expensify/App/assets/3069065/125e76d2-4bba-4f12-ace6-ee0b1f2af881

**Comment by mountiny** [comment]
Yeah I think the last thing is fine

**Comment by mananjadhav** [comment]
I thought earlier it was DistanceRequest. But then I also couldn’t figure out how and when it is rendered? 

I had DMed @hayata-suenaga too for distance request.

**Comment by rezkiy37** [comment]
Hi, guys! I was invited by Vit to test the PR as well 🙂

**Comment by mountiny** [comment]
@parasharrajat what slack thread exactly, are you still waiting on some information confirmation?

File: src/components/MoneyRequestSkeletonView.js [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+                foregroundColor={themeColors.border}
+            >
+                <Rect
+                    x="16"
+                    y="20"
+                    width="40"
+                    height="8"
+                />
+                <Rect
+                    x="15"
```
[the remark this answers is not in the document]
Comment (mananjadhav) [reply src/components/MoneyRequestSkeletonView.js]: @rezkiy37 Good eye. Corrected.

**Comment by rezkiy37** [comment]
> I didn't understand the issue. What is the problem here?

Take a look at the entire message background color and money request background. They are the same, no contrast.

Screenshot 2023-09-04 at 12 24 02

**Comment by mountiny** [comment]
@rezkiy37 good find, Manan is now afk but he will be back in an hour. Could you make a suggestion in code which should fix it to make his work faster once he comes back?

I think that in whisper it should look like when hovered and when hovered, no style change as we had in another case

**Comment by shawnborton** [comment]
I reported some design weirdness not related to the skeleton [here](https://expensify.slack.com/archives/C057F53HR40/p1693830657441109).

In terms of the skeleton itself, can we make sure it has the same pulsing effect as other skeleton loaders throughout the app too? 

Also just cc @dannymcclain on this issue since I think he worked on the mockups :)

File: src/components/ReportActionItem/MoneyRequestPreview.js [hunk 7]
```diff
@@ -227,81 +228,85 @@ function MoneyRequestPreview(props) {
                             isHovered={isScanning}
                         />
                     )}
-                    <View style={styles.moneyRequestPreviewBoxText}>
```
[the remark this answers is not in the document]
Comment (mananjadhav) [reply src/components/ReportActionItem/MoneyRequestPreview.js:237]: No nothing is changed. Why?

**Comment by mountiny** [comment]
@parasharrajat @mananjadhav on eis the same with the whisper

the other one is the bacground colour border
image

**Comment by mananjadhav** [comment]
The shimmer (pulsing) effect is showing when I add a certain width to it.

https://github.com/Expensify/App/assets/3069065/12889e86-e4f6-4e55-94dc-ed318df989f1

**Comment by rezkiy37** [comment]
> @rezkiy37 good find, Manan is now afk but he will be back in an hour. Could you make a suggestion in code which should fix it to make his work faster once he comes back?
> 
> I think that in whisper it should look like when hovered and when hovered, no style change as we had in another case

That's the commit how I fixed it - https://github.com/Expensify/App/pull/26603/commits/8bcad7cde66b3e074ec3d2189d8eba891ccf7b64.

**Comment by mountiny** [comment]
Lets address those and the whisper hover and we can ship it

**Comment by rezkiy37** [comment]
> @rezkiy37 Thanks for reporting the issue, but for some reason I am not able to reproduce this issue at my end. Can you tell me the reproduction steps?
> 
>  web-whisper-hover-style.mov

Yeah, I see. Let try one more time, maybe someone has fixed it right now 😅

**Comment by rezkiy37** [comment]
Hah, I was fixed in another [PR](https://github.com/Expensify/App/pull/26603) that was merged 3 hours ago. No bug anymore 🙂

**Comment by mountiny** [comment]
`vit-26503theft` to the rescue

**Comment by mananjadhav** [comment]
@mountiny @parasharrajat @rezkiy37 do we have any more outstanding comments?

**Comment by rezkiy37** [comment]
> @mountiny @parasharrajat @rezkiy37 do we have any more outstanding comments?

From my side, LGTM!

**Review by rezkiy37 — APPROVED** [review]

**Review by parasharrajat — APPROVED** [review]
## Reviewer Checklist

- [x] I have verified the author checklist is complete (all boxes are checked off).
- [x] I verified the correct issue is linked in the `### Fixed Issues` section above
- [x] I verified testing steps are clear and they cover the changes made in this PR
    - [x] I verified the steps for local testing are in the `Tests` section
    - [x] I verified the steps for Staging and/or Production testing are in the `QA steps` section
    - [x] I verified the steps cover any possible failure scenarios (i.e. verify an input displays the correct error message if the entered data is not correct)
    - [x] I turned off my network connection and tested it while offline to ensure it matches the expected behavior (i.e. verify the default avatar icon is displayed if app is offline)
- [x] I checked that screenshots or videos are included for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I included screenshots or videos for tests on [all platforms](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#make-sure-you-can-test-on-all-platforms)
- [x] I verified tests pass on **all platforms** & I tested again on:
    - [x] Android / native
    - [x] Android / Chrome
    - [x] iOS / native
    - [x] iOS / Safari
    - [x] MacOS / Chrome / Safari
    - [x] MacOS / Desktop
- [x] If there are any errors in the console that are unrelated to this PR, I either fixed them (preferred) or linked to where I reported them in Slack
- [x] I verified proper code patterns were followed (see [Reviewing the code](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md#reviewing-the-code))
    - [x] I verified that any callback methods that were added or modified are named for what the method does and never what callback they handle (i.e. `toggleReport` and not `onIconClick`).
    - [x] I verified that the left part of a conditional rendering a React component is a boolean and NOT a string, e.g. `myBool && <MyComponent />`.
    - [x] I verified that comments were added to code that is not self explanatory
    - [x] I verified that any new or modified comments were clear, correct English, and explained "why" the code was doing something instead of only explaining "what" the code was doing.
    - [x] I verified any copy / text shown in the product is localized by adding it to `src/languages/*` files and using the [translation method](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60)
    - [x] I verified all numbers, amounts, dates and phone numbers shown in the product are using the [localization methods](https://github.com/Expensify/App/blob/4bd99402cebdf4d7394e0d1f260879ea238197eb/src/components/withLocalize.js#L60-L68)
    - [x] I verified any copy / text that was added to the app is grammatically correct in English. It adheres to proper capitalization guidelines (note: only the first word of header/labels should be capitalized), and is approved by marketing by adding the `Waiting for Copy` label for a copy review on the original GH to get the correct copy.
    - [x] I verified proper file naming conventions were followed for any new files or renamed files. All non-platform specific files are named after what they export and are not named "index.js". All platform-specific files are named for the platform the code supports as outlined in the README.
    - [x] I verified the JSDocs style guidelines (in [`STYLE.md`](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs)) were followed
- [x] If a new code pattern is added I verified it was agreed to be used by multiple Expensify engineers
- [x] I verified that this PR follows the guidelines as stated in the [Review Guidelines](https://github.com/Expensify/App/blob/main/contributingGuides/PR_REVIEW_GUIDELINES.md)
- [x] I verified other components that can be impacted by these changes have been tested, and I retested again (i.e. if the PR modifies a shared library or component like `Avatar`, I verified the components using `Avatar` have been tested & I retested again)
- [x] I verified all code is DRY (the PR doesn't include any logic written more than once, with the exception of tests)
- [x] I verified any variables that can be defined as constants (ie. in CONST.js or at the top of the file that uses the constant) are defined as such
- [x] If a new component is created I verified that:
    - [x] A similar component doesn't exist in the codebase
    - [x] All props are defined accurately and each prop has a `/** comment above it */`
    - [x] The file is named correctly
    - [x] The component has a clear name that is non-ambiguous and the purpose of the component can be inferred from the name alone
    - [x] The only data being stored in the state is data necessary for rendering and nothing else
    - [x] For Class Components, any internal methods passed to components event handlers are bound to `this` properly so there are no scoping issues (i.e. for `onClick={this.submit}` the method `this.submit` should be bound to `this` in the constructor)
    - [x] Any internal methods bound to `this` are necessary to be bound (i.e. avoid `this.submit = this.submit.bind(this);` if `this.submit` is never passed to a component event handler like `onClick`)
    - [x] All JSX used for rendering exists in the render method
    - [x] The component has the minimum amount of code necessary for its purpose, and it is broken down into smaller components in order to separate concerns and functions
- [x] If any new file was added I verified that:
    - [x] The file has a description of what it does and/or why is needed at the top of the file if the code is not self explanatory
- [x] If a new CSS style is added I verified that:
    - [x] A similar style doesn't already exist
    - [x] The style can't be created with an existing [StyleUtils](https://github.com/Expensify/App/blob/main/src/styles/StyleUtils.js) function (i.e. `StyleUtils.getBackgroundAndBorderStyle(themeColors.componentBG`)
- [x] If the PR modifies code that runs when editing or sending messages, I tested and verified there is no unexpected behavior for all supported markdown - URLs, single line code, code blocks, quotes, headings, bold, strikethrough, and italic.
- [x] If the PR modifies a generic component, I tested and verified that those changes do not break usages of that component in the rest of the App (i.e. if a shared library or component like `Avatar` is modified, I verified that `Avatar` is working as expected in all cases)
- [x] If the PR modifies a component related to any of the existing Storybook stories, I tested and verified all stories for that component are still working as expected.
- [x] If a new page is added, I verified it's using the `ScrollView` component to make it scrollable when more elements are added to the page.
- [x] If the `main` branch was merged into this PR after a review, I tested again and verified the outcome was still expected according to the `Test` steps.
- [x] I have checked off every checkbox in the PR reviewer checklist, including those that don't apply to this PR.


:ribbon: :eyes: :ribbon:  C+ reviewed

**Review by mountiny — APPROVED** [review]
Thanks for all the hard work!

File: src/components/ReportActionItem/MoneyRequestPreview.js [hunk 8]
```diff
@@ -234,77 +235,81 @@ function MoneyRequestPreview(props) {
...
+                            <View style={[styles.flexRow]}>
+                                <View style={[styles.flex1, styles.flexRow, styles.alignItemsCenter]}>
+                                    <Text style={[styles.textLabelSupporting, styles.mb1, styles.lh20]}>{getPreviewHeaderText()}</Text>
+                                    {Boolean(getSettledMessage()) && (
+                                        <>
+                                            <Icon
+                                                src={Expensicons.DotIndicator}
+                                                width={4}
+                                                height={4}
+                                                additionalStyles={[styles.mr1, styles.ml1]}
```
Comment (rushatgabhane) [inline src/components/ReportActionItem/MoneyRequestPreview.js:251]: Coming from [Dot separator in the report preview component appears bottom aligned](https://github.com/Expensify/App/issues/29543#top)
#29543

`styles.mb1` isn't passed to `Icon`.

**Pull request merged**